### PR TITLE
Added Webiny to the list

### PR DESCRIPTION
**Pull request by avats-dev**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](../CONTRIBUTING.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

**Comment by avats-dev** [comment]
@crweiner [this](https://dev.to/webiny/webiny-hacktoberfest-2020-49h4) doesn't provides enough info on how webiny will contact people and what are their requirements in a strict sense, so I've added how I've best understood the blog. But you are welcome to take a read and review this. Thanks :tada:

File: README.md [hunk 1]
```diff
@@ -85,6 +85,16 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: Fill the [form](https://forms.gle/aUYJXoVJVivPgGuM8) and make a successful PR on any of [LoginRadius repository](https://github.com/LoginRadius).
 - **Notes**: Keep in touch from our [LoginRadius Engineering Blog](https://www.loginradius.com/engineering/page/hacktoberfest2020) site for more updates.
 
+### W
+
+#### **Webiny**
+
+- **Swag**: Special Webiny Swag Pack (T-Shirt)
+- **Requirements**: Make four successful pull requests in October 2020 to any open source project and atleast one pull request to [Webiny](https://github.com/webiny/webiny-js)
```
Comment (hugovk) [inline README.md:93]: Little typo:
```suggestion
- **Requirements**: Make four successful pull requests in October 2020 to any open source project and at least one pull request to [Webiny](https://github.com/webiny/webiny-js)
```

**Comment by crweiner** [comment]
Humm....good point @avats-dev, it really isn't all that clear. I [sent them a tweet](https://twitter.com/C_Dubbs/status/1311522318582243328) so let's see if we can get it all sorted out. Thanks for getting it started! 😄

**Comment by Rugved-HACKER** [comment]
very good work done

**Comment by AlbionaHoti** [comment]
> Humm....good point @avats-dev, it really isn't all that clear. I [sent them a tweet](https://twitter.com/C_Dubbs/status/1311522318582243328) so let's see if we can get it all sorted out. Thanks for getting it started! 😄

Hi @crweiner :) Thanks for sending the tweet. We updated the [blog](https://dev.to/webiny/webiny-hacktoberfest-2020-49h4). Let us know if it's all good!

@avats-dev thanks for jumping in and adding Webiny on the list 🚀

**Comment by crweiner** [comment]
Thanks so much for the clarification @AlbionaHoti! 👍 Two merged PRs sound great!

@avats-dev, can you please fix the merge conflicts and add Webiny to the proper spot in the "Least Involvement to Most Involvement" section following the [Contributing](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/CONTRIBUTING.md) guidelines? Thanks!

**Comment by avats-dev** [comment]
Fixing these and updating .... :rocket:

**Comment by crweiner** [comment]
> Fixing these and updating .... 🚀

Great, thank you very much! Let me know if you need help with the merge conflict and when you've added them to the "Least Involvement to Most Involvement" section of the List. 👍

**Comment by avats-dev** [comment]
@crweiner done :tada:

**Comment by avats-dev** [comment]
@crweiner can you take a look?

**Comment by crweiner** [comment]
Hey @avats-dev! Thank you very much for your great contribution and adding Webiny to the 2020 Hacktoberfest Swag List! I appreciate you taking the time to not only submit this PR, but keep working on it and fixing it. 🎉 🎉 

Also, thank you to @AlbionaHoti for the PR clarifications and participating in Hacktoberfest! It's been quite an interesting few days to say the least. 😅 

And of course, the ever amazing @hugovk for his review and mentoring. I don't know how you find the time in the day to be such an amazing help across endless repos! Thank you so much for your hard work. You're a legend. 🤓 

you're awesome

**Pull request merged**